frontend/packages/shared/src/components/altinnHeaderButtons/AltinnHeaderButtons.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classes from './AltinnHeaderbuttons.module.css';
+import { Button, ButtonVariant } from '@digdir/design-system-react';
+import { useTranslation } from 'react-i18next';
+import { previewPath, publiserPath } from 'app-shared/api-paths';
+import { useParams } from 'react-router-dom';
+import { TopBarMenu } from 'app-development/layout/AppBar/appBarConfig';
+
+export const AltinnHeaderButtons = () => {
```

Comment:
This is still app-development specific, we want a generic header component. Suggest that this component is either passed as a prop to the header component entirely, or made generic so that it can take in a list of actions (with titles) and generate the buttons based on those.

Reply:
used generic with list of actions